1-js/02-first-steps/06-alert-prompt-confirm/article.md

```diff
@@ -1,6 +1,6 @@
 # Interação: alert, prompt, confirm
 
-Como usuaremos o navegador como nosso ambiente de demonstração, vamos ver algumas funções para interagir com o usuário: `alert`, `prompt` e `confirm`.
+Como usaremos o navegador como nosso ambiente de demonstração, vamos ver algumas funções para interagir com o usuário: `alert`, `prompt` e `confirm`.
 
 ## alert
 
```

Comment:
```suggestion
Esta já vimos. Ela mostra uma mensagem e aguarda que o usuário pressione "OK".
```

Reply:
Done in https://github.com/javascript-tutorial/pt.javascript.info/pull/244/commits/e072d452a5d1a24871c5bf9e1fba0ff81ad4a6a5